common/assets/scss/application.scss

```diff
@@ -29,6 +29,9 @@ $govuk-font-family: "Inter", system-ui, sans-serif;
 @import "govuk-frontend/govuk/utilities/all";
 @import "govuk-frontend/govuk/overrides/all";
 
+// Import MoJ Frontend Components
+@import "@ministryofjustice/frontend/moj/components/organisation-switcher/_organisation-switcher.scss";
```

Comment:
Is there a way to add this module to the sass load paths to reduce the amount of folders needed. Similar to how the GOV.UK frontend is loaded.

Reply:
I think this is being loaded in the same way, just the path in node_modules is longer in this library.

Reply:
Ah yes. Tad annoying they've added that `frontend/` folder. Not sure what its use is...